packages/shared/src/components/checklist/ChecklistStep.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { IconSize } from '../Icon';
+import ArrowIcon from '../icons/Arrow';
+import { ChecklistStepProps } from '../../lib/checklist';
+import { QuaternaryButton } from '../buttons/QuaternaryButton';
+import ChecklistAIcon from '../icons/ChecklistA';
+
+const ChecklistStep = ({
+  className = {},
+  step,
+  checked = false,
+  active = false,
+  onToggle,
+  children,
+}: ChecklistStepProps): ReactElement => {
+  const isCompleted = !!step.action.completedAt;
+  const isOpen = !isCompleted && checked;
+
+  return (
+    <div className={className.root}>
+      <button
+        type="button"
+        disabled={isCompleted}
+        className="flex justify-between items-center w-full"
+        onClick={(event) => {
+          event.preventDefault();
+
+          onToggle(step.action);
+        }}
+      >
+        <div className="flex flex-1 gap-1 items-center flex-start">
+          <QuaternaryButton
```

Comment:
Why do we need the button inside a button?
Can't the outside button carry all the styling directly?

Reply:
I added this just so styles match what is in sketch (since it is that button there) outside one is a actually clickable handler. Let me know what would be better way to do it.

Reply:
I did have styling issues with text wrapping and positioning because `children` of `QuaternaryButton` are wrapped in `label` which also does not have styling exposed. 

I removed the `QuaternaryButton` in fcc3ad5635c3ed36edc3c1d63fca3dc36bb524fa and just used plain styles, it looks the same.